simtools/applications/simulate_prod.py

```diff
@@ -140,25 +139,22 @@ def _parse(description=None):
     )
     config.parser.add_argument(
         "--from_azimuth_direction",
-        help="Direction from which the primary reaches the atmosphere",
-        type=str.lower,
+        help=(
+            "Direction from which the primary reaches the atmosphere."
+            "It can be in degrees between 0 and 360 or one of (north, south, east, west)."
+        ),
+        type=CommandLineParser.azimuth_angle,
         required=True,
-        choices=[
-            "north",
-            "south",
-            "east",
-            "west",
-        ],
     )
     config.parser.add_argument(
         "--zenith_angle",
-        help="Zenith angle in degrees",
+        help="Zenith angle in degrees (between 0 and 180).",
```

Comment:
Do we allow showers coming from below? 180 deg?

Reply:
CORSIKA can do it and MAGIC sometimes does it, so I don't see a reason to limit it here.

Reply:
OK!